lib/beautiful/xresources.lua

```diff
@@ -3,7 +3,7 @@
 --
 -- @author Yauhen Kirylau &lt;yawghen@gmail.com&gt;
 -- @copyright 2015 Yauhen Kirylau
--- @module beautiful.xresources
+-- @submodule beautiful
```

Comment:
i've a bit underlooked, i suggest reverting this line, because xresources module have quite specific usage related to one particular service integration, not some common theming helpers, like theme_assets above

Reply:
Makes sense.

@Elv13 ?

Reply:
On one hand @actionless has a point. On the other, if we consolidate more pages, the end result is less or more the same. What about taking a look what prevents the `gtk` module from being consolidated and then how to better document these things. Right now the discoverability factor of the theme API is a rather big issue (plus the "wall of text effect" of out current documentation index).

IMHO, beside the (partially deprecated) DPI stuff, these APIs are only relevant to the xresources (based) theme(s). If there was XRDB write support and you could, for example, change urxvt theme dynamically and control compton using an extended version of this, then my position would be much for favorable to make this a top level again.